README.md

```diff
@@ -2,7 +2,7 @@ Elixometer
 ==========
 
 [![Build Status](https://travis-ci.org/pinterest/elixometer.svg?branch=master)](https://travis-ci.org/pinterest/elixometer)
-[![Coverage Status](https://coveralls.io/repos/pinterest/elixometer/badge.svg?branch=&service=github)](https://coveralls.io/github/pinterest/elixometer?branch=)
+[![Coverage Status](https://coveralls.io/repos/pinterest/elixometer/badge.svg?branch=&service=github)](https://coveralls.io/github/pinterest/elixometer?branch=master)
```

Comment:
Right now everything is master, so it may not matter.

If you decide to have a release branch for more stable code, 
then it is better to be more specific.